internal/admin/dashboard/static/css/dashboard.css

```
@@ -791,6 +859,204 @@ body {
+    border-radius: 50%;
+    flex-shrink: 0;
+}
+
+.donut-legend-label {
+    flex: 1;
+    min-width: 0;
+    overflow: hidden;
+    text-overflow: ellipsis;
+    white-space: nowrap;
+    color: var(--text);
+    font-family: 'SF Mono', 'Menlo', 'Consolas', monospace;
```

Stylelint (17.3.0)

[error] Unexpected quotes around "Menlo" (font-family-name-quotes)

(font-family-name-quotes)